OJP/OJP_FareSupport.xsd

```diff
@@ -8,6 +8,20 @@
 	<xs:annotation>
 		<xs:documentation>========================================== Fare Authority ==========================================</xs:documentation>
 	</xs:annotation>
+	<xs:complexType name="BookingNoteStructure">
+		<xs:annotation>
+			<xs:documentation>A structure for booking notes.</xs:documentation>
+		</xs:annotation>
+		<xs:sequence>
+			<xs:element name="BookingNote" type="BookingNoteType" minOccurs="1" maxOccurs="unbounded"/>
+		</xs:sequence>
+	</xs:complexType>
+	<xs:complexType name="BookingNoteType">
+		<xs:annotation>
+			<xs:documentation>An element of the BookingNotes or as a single BookingNote.</xs:documentation>
```

Comment:
I do not understand this documentation. Could it please be reworded?

Reply:
reworded